rust/noosphere-core/Cargo.toml

```diff
@@ -50,6 +54,8 @@ noosphere-collections = { version = "0.6.0", path = "../noosphere-collections" }
 
 ucan = { workspace = true }
 ucan-key-support = { workspace = true }
+sentry-tracing = { version = "0.31.5", optional = true }
+sentry = { version = "0.31.5", optional = true, default-features = false, features=["reqwest", "rustls", "backtrace", "contexts", "panic", "anyhow"] }
```

Comment:
Did we verify if this is enough to evade the OpenSSL dependency?

Reply:
yes.